apps/server/src/modules/user/legacy/repo/users-admin.repo.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { MongoMemoryDatabaseModule } from '@infra/database';
+import { User } from '@shared/domain/entity';
+import { UsersAdminRepo } from './users-admin.repo';
+
+describe('users admin repo', () => {
```

Comment:
Didn't you write more tests here, because they would basically be the same as the API tests?

Reply:
exactly

Reply:
IMO the right way to do it is to test all variations of the repo methods here. In the API tests I would not test all the variations of queries etc., just the most common cases. I'm not really sure what's the project's standard for it, but I guess it is not okay to just leave out the repo tests.